tubular/panel_backend_api.py

```diff
@@ -0,0 +1,26 @@
+import requests
+
+
+RETIREMENT_ENDPOINT = '/api/v1/retirement/'
+
+class PanelBackendAPI():
+    """
+    Panel Backend API to retire user data.
+    """
+
+    def __init__(self, panel_backend_base_url, panel_backend_auth_token):
+        self.base_url = panel_backend_base_url
+        self.auth_token = panel_backend_auth_token
+
+    def retire_user(self, learner):
+        """
+        Retires the user with the given username in the Edly Panel Backend.
+        """
+
+        url = "{}{}".format(self.base_url, RETIREMENT_ENDPOINT)
+        headers = {'Authorization': 'Token {}'.format(self.auth_token)}
+        data = {'username': learner['original_username']}
+
+        response = requests.post(url, data=data, headers=headers)
```

Comment:
what if there is an error in the post request? any exception handling mechanism added?

Reply:
Yes it will raise the exception https://github.com/edly-io/tubular/pull/3/files#diff-ac0759b0360762f86e722cd524329de2973c5889e6f633122a86008e55dd4c8cR25 if there any error and Tubular will handle reset of functionality